src/lib.rs

```diff
@@ -132,8 +131,30 @@ pub struct LoadWorld;
 #[derive(ScheduleLabel, Debug, Hash, PartialEq, Eq, Clone)]
 pub struct SaveWorld;
 
+/// Label for the schedule which advances the current world to the next frame.
+#[derive(ScheduleLabel, Debug, Hash, PartialEq, Eq, Clone)]
+pub struct AdvanceWorld;
+
 /// GGRS plugin for bevy.
 ///
+/// # Rollback
+///
+/// This will provide rollback management for the following items in the Bevy ECS:
+/// - [Entities](`Entity`)
+/// - [Parent] and [Children] components
+/// - [Time]
+///
+/// To add more data to the rollback management, see the methods provided by [GgrsApp].
+///
+/// # Rollback
+///
+/// This will provide rollback management for the following items in the Bevy ECS:
+/// - [Entities](`Entity`)
+/// - [Parent] and [Children] components
+/// - [Time]
+///
+/// To add more data to the rollback management, see the methods provided by [GgrsApp].
+///
```

Comment:
This comment is repeated twice.

Reply:
That looks like a casualty of my rebase, I'll fix that in a separate PR now. Thanks for spotting it!